src/routes/usersRoute.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const express = require('express');
+const services = require('../services/usersService');
+const cors = require('cors');
+
+const router = express.Router();
+
+router.use(cors());
+
+router.get('/', (req, res) => {
+  res.statusCode = 200;
+  res.send(services.getUsers());
+});
```

Comment:
it will be great to transfer logic from router, in controllers you could create function
```
function getAll(req, res) {
   const users = usersServise.getAll();

   res.statusCode = 200;
   res.send(users);
 }
```
and your route will look like this
`router.get('/', usersControllers.getAll);`

you also need to change other routes, then the architecture of your application will be correct